src/main/scala/com/raphtory/output/FileOutputFormat.scala

```diff
@@ -7,6 +7,19 @@ import org.slf4j.LoggerFactory
 
 import java.io.File
 
+/**
+  * # FileOutputFormat
+  * `FileOutputFormat` writes output for Raphtory Job and Partition for a pre-defined window and timestamp to File
+  *
+  * Usage while querying or running algorithmic tests:
+  *
+  * ```{code-block} scala
+  * import com.raphtory.output.FileOutputFormat
+  * algorithmTest(WeightedDegree(), outputFormat, 1, 32674, 10000, List(500, 1000, 10000))
+  * val testDir  =  "/tmp/raphtoryTest"
+  * val defaultOutputFormat: OutputFormat  =  FileOutputFormat(testDir)
+  * ```
+  */
```

Comment:
a ` ```{seealso}``` ` section would be nice pointing to the Raphtory Query API and the OutputFormat base class